common/src/main/java/com/wynntils/wynn/model/bossbar/BossBarModel.java

```diff
@@ -4,27 +4,19 @@
  */
 package com.wynntils.wynn.model.bossbar;
 
-import com.wynntils.core.WynntilsMod;
-import com.wynntils.core.managers.Managers;
+import com.wynntils.core.managers.Handlers;
 import com.wynntils.core.managers.Model;
-import com.wynntils.mc.event.BossHealthUpdateEvent;
-import com.wynntils.mc.event.CustomBarAddEvent;
-import com.wynntils.mc.utils.ComponentUtils;
-import com.wynntils.mc.utils.McUtils;
-import com.wynntils.wynn.objects.ClassType;
+import com.wynntils.handlers.bossbar.TrackedBar;
 import java.util.Arrays;
-import java.util.HashMap;
-import java.util.UUID;
-import java.util.function.Consumer;
-import java.util.regex.Matcher;
-import java.util.regex.Pattern;
-import net.minecraft.client.gui.components.LerpingBossEvent;
-import net.minecraft.network.chat.Component;
-import net.minecraft.network.protocol.game.ClientboundBossEventPacket;
-import net.minecraft.world.BossEvent;
-import net.minecraftforge.eventbus.api.SubscribeEvent;
+import java.util.List;
 
 public final class BossBarModel extends Model {
+    public static final String BLOODPOOL = "bloodpool";
+    public static final String MANABANK = "manabank";
+    public static final String AWAKENED = "awakened";
+    public static final String FOCUS = "focus";
+    public static final String CORRUPTED = "corrupted";
```

Comment:
Seems a bit weird. Can't we make this an enum? What is the use case?

Reply:
It used to be an enum, but I removed it. Here's the issue:
We need to make a connection between BossBars (models) and BossBarOverlays (feature). So when CustomBarsOverlayFeature gets a BossBarAddedEvent, it should create the corresponding overlay.

But, the old style made this by propagating a global enum throught the TrackedBar class. But this is mixing different abstraction levels, since now the BossBarHandler suddently gets a dependency on an enum listing known types of boss bars.

But I agree that this is not ideal either. I'll see if I can come up with something better.

Reply:
@kristofbolyai Thanks for the pushback. :) I was a bit lazy. It is now much better!